apps/src/applab/PhoneFrame.jsx

```diff
@@ -0,0 +1,69 @@
+import Radium from 'radium';
+import commonStyles from '../commonStyles';
+import color from '../color';
+import applabConstants from './constants';
+import experiments from '../experiments';
+
+const RADIUS = 30;
+
+const styles = {
+  phoneFrame: {
+    display: 'block',
+    height: 50,
+    backgroundColor: color.lighter_gray,
+  },
+  phoneFrameRunning: {
+    backgroundColor: color.charcoal
+  },
+  phoneFrameTop: {
+    borderTopLeftRadius: RADIUS,
+    borderTopRightRadius: RADIUS,
+  },
+  phoneFrameBottom: {
+    borderBottomLeftRadius: RADIUS,
+    borderBottomRightRadius: RADIUS,
+  },
+  nonResponsive: {
+    maxWidth: applabConstants.APP_WIDTH,
+  }
+};
+
+
+const PhoneFrame = React.createClass({
+  propTypes: {
+    showFrame: React.PropTypes.bool.isRequired,
+    isRunning: React.PropTypes.bool.isRequired
+  },
+
+  render: function () {
```

Comment:
IIRC they require some different testing patterns too.  I'm fine with this as-is.